dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/ShowClaimItemFeatureIT.java

```diff
@@ -0,0 +1,177 @@
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded").exists())
+                        .andExpect(jsonPath("$.page.totalElements", greaterThanOrEqualTo(1)));
+
+        getClient(token).perform(get("/api/authz/authorizations/search/object")
+                                     .param("uri", uri(collectionBProfile))
+                                     .param("eperson", context.getCurrentUser().getID().toString())
+                                     .param("feature", showClaimProfileFeature.getName()))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded").exists())
+                        .andExpect(jsonPath("$.page.totalElements", greaterThanOrEqualTo(1)));
+
+    }
+
+    @Test
+    public void testNotClaimableEntity() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        Collection publicationCollection = CollectionBuilder
+            .createCollection(context, parentCommunity)
+            .withEntityType("Publication")
+            .withName("notClaimable")
+            .build();
+
+        context.turnOffAuthorisationSystem();
+
+        Item publication = ItemBuilder.createItem(context, publicationCollection).build();
+
+        String token = getAuthToken(context.getCurrentUser().getEmail(), password);
+
+        getClient(token).perform(get("/api/authz/authorizations/search/object")
+                                     .param("uri", uri(publication))
+                                     .param("eperson", context.getCurrentUser().getID().toString())
+                                     .param("feature", showClaimProfileFeature.getName()))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded").doesNotExist())
+                        .andExpect(jsonPath("$.page.totalElements", equalTo(0)));
+
+    }
+
+    @Test
+    public void testItemAlreadyInARelation() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        Item ownedItem = ItemBuilder.createItem(context, personCollection)
+                                    .withDspaceObjectOwner("owner", "ownerAuthority").build();
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(context.getCurrentUser().getEmail(), password);
+
+        getClient(token).perform(get("/api/authz/authorizations/search/object")
+                                     .param("uri", uri(ownedItem))
+                                     .param("eperson", context.getCurrentUser().getID().toString())
+                                     .param("feature", showClaimProfileFeature.getName()))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded").exists())
+                        .andExpect(jsonPath("$.page.totalElements", greaterThanOrEqualTo(1)));
```

Comment:
Why do we want to show the "claim" button if the Item is already claimed?  Maybe this is what I don't understand about why we need both a ShowClaim and a CanClaim feature.  

(This test appears to be the only difference between ShowClaim & CanClaim...that's why this question is important.)

Reply:
@LucaGiamminonni : following up on this. I think it's very odd behavior to show a "Claim" button if you cannot actually claim something.  This is why I find this difference between "ShowClaim" and "CanClaim" so odd.  It seems like these two Features should really be checking for the same things (whether you are able to claim the Item). 

If you can claim the item, then the button should appear.  If you cannot, it should not appear.  

It doesn't make sense to be why you'd want the button to appear when an Item is already claimed by someone else (which is what "ShowClaim" currently allows as proven by this test).  As clicking the button will just result in an error from "CanClaim".

I simply don't understand the use case here & why the code is implemented in this manner.